spec/person_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../person'
+
+describe Person do
+  before :each do
+    @person = Person.new 1, 15, 'Kennedy', true
+  end
+  context 'When creating person' do
+    it 'create a person information' do
+      expect(@person).to be_an_instance_of Person
+    end
+    it 'can_use_services when parent permission == true' do
+      expect(@person.can_use_services?).to eql true
+    end
+    it "Name should be 'Kennedy'" do
+      expect(@person.name).to eql 'Kennedy'
+    end
+  end
+end
```

Comment:
- [ ] Based on the requirement, you are expected to test all the methods in all classes. In your Person_spec.rb file, i cannot see the test for `Person.add_rental` method. 

```ruby
def add_rental(date, book)
    Rental.new(date, book, self)
  end
```

- [ ] Also, the method `can_use_services?` test should cover all scenario, 👇🏾 
    - Negative scenario - The `can_use_services?` method returns false if under age and no parent permission
    - Positive Scenario - The `can_use_services?` method returns true if over age
    - The can_use_services? method returns true if has parent permission - **_You have implemented this_**
Please implement that.

Reply:
Thanks for the feedback. All suggestions implemented